From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Tue, 15 Sep 2020 01:32:24 +0200
Subject: tests: Disable tests requiring system bus

Parsing recursive dir test may require UDiskd and this needs to run in
the system bus, so disable it for now.
---
 plparse/tests/parser.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/plparse/tests/parser.c b/plparse/tests/parser.c
index 35f62f2..e4c7c03 100644
--- a/plparse/tests/parser.c
+++ b/plparse/tests/parser.c
@@ -1921,7 +1921,8 @@ main (int argc, char *argv[])
 		g_test_add_func ("/parser/parsing/m3u_leading_tabs", test_m3u_leading_tabs);
 		g_test_add_func ("/parser/parsing/empty-asx.asx", test_empty_asx);
 		g_test_add_func ("/parser/parsing/emptyplaylist.pls", test_empty_pls);
-		g_test_add_func ("/parser/parsing/dir_recurse", test_directory_recurse);
+		/* This requires udiskd, that needs the system bus */
+		// g_test_add_func ("/parser/parsing/dir_recurse", test_directory_recurse);
 		g_test_add_func ("/parser/parsing/async_signal_order", test_async_parsing_signal_order);
 		g_test_add_func ("/parser/parsing/wma_asf", test_parsing_wma_asf);
 		g_test_add_func ("/parser/parsing/remote_mp3", test_parsing_remote_mp3);
